st2reactor/st2reactor/container/sensor_wrapper.py

```diff
@@ -328,7 +328,9 @@ def __init__(self, pack, file_path, class_name, trigger_types,
                                                update_handler=self._handle_update_trigger,
                                                delete_handler=self._handle_delete_trigger,
                                                trigger_types=self._trigger_types,
-                                               queue_suffix='sensorwrapper')
+                                               queue_suffix='sensorwrapper_%s_%s' %
+                                               (self._pack, self._class_name),
+                                               exclusive=True)
```

Comment:
Yes I agree with using separate routing keys. I almost wanted to go with that right but too many changes - we will have to accept another messaging setup migration at a later point.